doctoshotgun.py

```diff
@@ -376,8 +383,8 @@ def main(self):
             docto.patient = patients[0]
 
         while True:
-            for center in docto.find_centers(args.city):
-                if center['city'].lower() != args.city:
+            for center in docto.find_centers(docto.remove_accents(args.city)):
+                if docto.remove_accents(center['city'].lower()) != docto.normalize(args.city):
```

Comment:
You haven't changed the name from `remove_accents` to `normalize` here

Reply:
Corrected on https://github.com/rbignon/doctoshotgun/pull/19/commits/c208be07de845f47f5d6431352a6571007ae1de2